opengrok-indexer/src/main/java/org/opengrok/indexer/history/HistoryGuru.java

```diff
@@ -540,39 +536,6 @@ public void updateRepositories() {
         }
     }
 
-    /**
-     * Update the source contents in given repositories.
-     *
-     * @param paths A list of files/directories to update
-     */
-    public void updateRepositories(Collection<String> paths) {
```

Comment:
how is this removal related to the repository scan ?

Reply:
While working in `HistoryGuru`, IDEA pointed out unused methods, and as we don't reflect into that class, it seemed a valid analysis — so I accepted to delete the methods.

Reply:
Yep, definitely dead code. Better do this in separate changeset. There might be an issue tracking the removal but I cannot find it at the moment. Also, the individual repositories need to be cleaned up, e.g. `MercurialRepository#update()` etc.

Reply:
OK undone

Reply:
this is tracked by #2330